# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../tomsfastmath/pdflatex.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./makefile.orig	2006-12-02 01:24:13.000000000 -0300
+++ ./makefile	2006-12-02 01:24:39.000000000 -0300
@@ -111,8 +111,7 @@
 	latex tfm >/dev/null
 
 docs: docdvi
-	latex tfm >/dev/null
-	dvipdf tfm
+	pdflatex tfm >/dev/null
 	mv -f tfm.pdf doc
 
 #This rule cleans the source tree of all compiled code, not including the pdf
